.github/workflows/update-data.yml

```diff
@@ -17,20 +17,24 @@ jobs:
     steps:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
 
+      - name: Enable corepack
+        shell: bash
+        run: corepack enable
+
       - name: Set up Node.js ${{ env.NODE_VERSION }}
         uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
         with:
           node-version: ${{ env.NODE_VERSION }}
-          cache: yarn
+          cache: pnpm
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: pnpm install
```

Comment:
```suggestion
        run: pnpm install --frozen-lockfile
```

Reply:
not necessary, it's default on CI, like with yarn v2+ 🙃

Reply:
https://pnpm.io/cli/install#--frozen-lockfile